hc/api/tests/test_ensuretriggers.py

```diff
@@ -13,17 +13,18 @@ def test_ensure_triggers(self):
         Command().handle()
 
         check = Check.objects.create()
-        assert check.alert_after is None
+        self.assertEqual(check.alert_after, None)
 
         check.last_ping = timezone.now()
         check.save()
         check.refresh_from_db()
-        assert check.alert_after is not None
-        ### The above assert fails. Make it pass
+        self.assertNotEqual(check.alert_after, None)
```

Comment:
We should use `assertIsNotNone` here.
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone